setup/setup_all.sh

```diff
@@ -102,10 +102,15 @@ init_foundation() {
   
   # Enabling GCP services first.
   terraform apply -target=module.project_services -target=module.service_accounts -auto-approve
-  
-  # Initializing Firebase (Only need this for the first time.)
-  # NOTE: the Firebase can only be initialized once (via App Engine).
-  terraform apply -target=module.firebase -var="firebase_init=true" -auto-approve
+
+  # Check if firebase is already initialized
+  KEY_NAME=$(gcloud alpha services api-keys list --filter="displayName='Browser key (auto created by Firebase)'" \
```

Comment:
I double-checked this method, this is not reliable. The API key will only be created when creating a Firebase app on the Console UI.

Reply:
I will add a TODO comment to change this with a more reliable method